content/en/data_streams/dotnet.md

```diff
@@ -8,13 +8,14 @@ title: Setup Data Streams Monitoring for .NET
 
 ### Supported libraries
 
-| Technology | Library                                                                        | Minimal tracer version | Recommended tracer version |
-|------------|--------------------------------------------------------------------------------|------------------------|----------------------------|
-| Kafka      | [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka)              | 2.28.0                 | 2.41.0 or later            |
-| RabbitMQ   | [RabbitMQ.Client](https://www.nuget.org/packages/RabbitMQ.Client)              | 2.28.0                 | 2.37.0 or later            |
-| Amazon SQS | [AWS SQS SDK](https://www.nuget.org/packages/AWSSDK.SQS)                       | 2.48.0                 | 2.48.0 or later            |
-| Amazon SNS | [AWS SNS SDK](https://www.nuget.org/packages/AWSSDK.SimpleNotificationService) | 3.6.0                  | 3.6.0 or later             |
-| IBM MQ     | [IBMMQDotnetClient](https://www.nuget.org/packages/IBMMQDotnetClient)          | 2.49.0                 | 2.49.0 or later            |
+| Technology        | Library                                                                                 | Minimal tracer version | Recommended tracer version |
+|-------------------|-----------------------------------------------------------------------------------------|------------------------|----------------------------|
+| Kafka             | [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka)                       | 2.28.0                 | 2.41.0 or later            |
```

Comment:
Could you update the links in the table so they're not inline? (For context, our translation process requires us to use numbered destination links.)

For example, instead of having the Confluent.Kafka link inline, it would instead be formatted to looks something like:

[Confluent.Kafka][1]
 
And at the bottom of the .md file, you would have the destination link:

[1]: https://www.nuget.org/packages/Confluent.Kafka

Reply:
Makes sense, I updated the links.